ej2-react/appearance/icons.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Icons in React Appearance component | Syncfusion
-description: Learn here all about Icons in Syncfusion React Appearance component of Syncfusion Essential JS 2 and more.
+title: Icons in React Appearance Component | Syncfusion
```

Comment:
Icons in React Appearance | Syncfusion - Change like this

Reply:
comment resolved